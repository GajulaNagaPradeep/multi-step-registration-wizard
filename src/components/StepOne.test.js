import React from "react";
import { shallow } from "enzyme";
import StepOne from "./StepOne";

const props = {
  user: {
    first_name: "name",
    last_name: "",
    user_email: "",
    user_password: "",
  },
};

let wrapped = shallow(<StepOne {...props} />);
describe("Progress", () => {
  it("should be rendered correctly", () => {
    expect(wrapped).toMatchSnapshot();
  });
});
