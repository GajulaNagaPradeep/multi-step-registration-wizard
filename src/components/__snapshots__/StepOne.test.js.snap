// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress should be rendered correctly 1`] = `
<Form
  className="input-form"
  inline={false}
>
  <ForwardRef(MotionComponent)
    animate={
      Object {
        "x": 0,
      }
    }
    className="col-lg-6 offset-lg-3"
    initial={
      Object {
        "x": "-100vw",
      }
    }
    transition={
      Object {
        "stiffness": 150,
      }
    }
  >
    <FormGroup
      controlId="first_name"
    >
      <FormLabel
        column={false}
        srOnly={false}
      >
        First Name
      </FormLabel>
      <FormControl
        autoComplete="off"
        className=""
        name="first_name"
        placeholder="Enter your first name"
        type="text"
      />
    </FormGroup>
    <FormGroup
      controlId="last_name"
    >
      <FormLabel
        column={false}
        srOnly={false}
      >
        Last Name
      </FormLabel>
      <FormControl
        autoComplete="off"
        className=""
        name="last_name"
        placeholder="Enter your last name"
        type="text"
      />
    </FormGroup>
  </ForwardRef(MotionComponent)>
  <Button
    active={false}
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "float": "right",
      }
    }
    type="submit"
    variant="primary"
  >
    Next
  </Button>
</Form>
`;
